src/modules/buttons/Button.jsx

```diff
@@ -6,18 +6,29 @@ import './Button.css';
 
 const propTypes = {
   label: PropTypes.string,
+  mod: PropTypes.string,
```

Comment:
@Vxplus I think `type` isn't best choice also (`type="submit"`). But in general I've got your idea